src/PlaidLink.js

```diff
@@ -71,7 +71,7 @@ const PlaidLink = React.createClass({
     buttonText: React.PropTypes.string,
 
     // Button Styles as an Object
-    styles: React.PropTypes.object,
+    buttonStyles: React.PropTypes.object,
```

Comment:
I think we should just rename this to `style` so that it mimics the standard style element prop